lib/Event/ReportEvents.php

```diff
@@ -14,16 +14,16 @@
  *  @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-namespace Pimcore\Event\Admin;
+namespace Pimcore\Event;
 
 final class ReportEvents
 {
     /**
      * The SAVE_SETTINGS event is triggered when reports settings are saved
      *
-     * @Event("Pimcore\Event\Admin\Report\SettingsEvent")
+     * @Event("Pimcore\Event\Report\SettingsEvent")
      *
      * @var string
      */
-    const SAVE_SETTINGS = 'pimcore.admin.reports.save_settings';
+    const SAVE_SETTINGS = 'pimcore.reports.save_settings';
```

Comment:
We should mention this somewhere, that the name changed. You have to change your Listener accordingly

Reply:
Added upgrade notes.